backend/index.js

```diff
@@ -100,21 +100,18 @@ app.use((req, res, next) => {
   req.session.nowInMinutes = Math.floor(Date.now() / 60e3)
   next()
 })
-app.use(setupWebpackForDev())
 // Extract pagination header information from requests and set on the 'context'
 app.use('/api', requestForwarding.extractRequestPaginationMiddleware)
-
 app.use(routes({ ...apis }))
 
+app.use(setupWebpackForDev())
```

Comment:
setting up webpack for dev needs to go after nunjucks pages otherwise it hijacks all of them